source/includes/_send_whatsapp.md

```diff
@@ -0,0 +1,244 @@
+# Send WhatsApp COMINGSOON
+
+OY's API Collection for sending Whatsapp Notifications.
```

Comment:
OY's API Collection for sending Whatsapp Notifications. Currently, WatsApp can only be enabled for Payment Link and Invoice product. Contact your Business Representative if you'd like to activate this feature.